### PR TITLE
chore: updated documentations to include code examples

### PR DESCRIPTION
**Pull request by oshinongit**
First take on how autogen documentation can look with code examples

### CODE REVIEW

**Review by birme — APPROVED** [review]
Looks good, just thinking though whether those that you marked as hidden actually should be private methods. But we can change that in another PR.

**Review by bwallberg — CHANGES_REQUESTED** [review]
The indentation is wrong sometimes, the `/**` first star should align with the bottom star `*/` ( like here: https://typedoc.org/tags/example/ ) 

Also every row of a comment should begin with `*`

File: packages/sdk/src/index.ts [hunk 1]
```diff
@@ -316,6 +341,16 @@ export class WHIPClient extends EventEmitter {
     }
   }
 
+  /**
+ * @example
+ * const videoIngest = document.querySelector<HTMLVideoElement>("video#ingest");
+ *const mediaStream = await navigator.mediaDevices.getUserMedia({
+  video: true,
```
Comment (bwallberg) [inline packages/sdk/src/index.ts:348]: missing `*`

File: packages/sdk/src/index.ts [hunk 2]
```diff
@@ -316,6 +341,16 @@ export class WHIPClient extends EventEmitter {
     }
   }
 
+  /**
+ * @example
+ * const videoIngest = document.querySelector<HTMLVideoElement>("video#ingest");
+ *const mediaStream = await navigator.mediaDevices.getUserMedia({
```
Comment (bwallberg) [inline packages/sdk/src/index.ts:347]: missing space between `*` and `c`

**Review by bwallberg — APPROVED** [review]
LGTM!

**Pull request merged**